docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -170,3 +170,61 @@ Result:
     ]]                                                                                                                   │
 └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## seriesDecomposeAnomaliesDetection
+
+Detect anomalies in series data using [series decomposition](#seriesDecomposeSTL).
```

Comment:
Please add one or two more sentences what happens under the hood. The reader understands that the function performs series decomposition but what happens then?